andela/Ah-frontend-guardians
# #162685624: Users should be able to reset their passwords

**What does this PR do?**
This PR enables a user to reset their password in case they forget it.

**Description of Task to be completed?**
A user should receive a URL in the email if they choose to change their password and then enter their new password.

**What are the relevant Pivotal Tracker stories?**
[#162685624](https://www.pivotaltracker.com/story/show/162685624)

**How to test it manually**
Pull the backend project using this command 

- Set up the backend project using these commands in your terminal.
```
$ git clone https://github.com/andela/Ah-backend-guardians.git
$ virtualenv venv
$ . venv/bin/activate
$ pip install -r requirements.txt
$ python manage.py runserver
```
- Setup the front-end project using these commands in your terminal.
```
$ git clone https://github.com/andela/Ah-frontend-guardians.git
$ npm install
$ npm start
```
- In your browser, visit the registration page and register a user.
- Log that user in successfully from the login page and then revert back to the `login page`.
- Click the `Forgot password?` link and enter the email for the registered user whose password you wish to change.
- Click the link in the email and enter the new password twice before you hit enter.
- Log the user in with the new password and for proof, you could attempt to log them in with the old password as well, which should fail.
